movable.lua

```diff
@@ -354,15 +354,17 @@ do
 
 	function frame:VARIABLES_LOADED()
 		-- Catch all oUF instances
-		for index = 1, GetNumAddOns() do
-			local addon = GetAddOnInfo(index)
-			local global = GetAddOnMetadata(addon, 'X-oUF')
-			if(global) then
-				table.insert(instances, _G[global])
+		if(#instances == 0) then
+			for index = 1, GetNumAddOns() do
+				local addon = GetAddOnInfo(index)
+				local global = GetAddOnMetadata(addon, 'X-oUF')
+				if(global) then
+					table.insert(instances, _G[global])
+				end
```

Comment:
Might be better to just check if _NAME differs from oUF: MovableFrames, not everyone might care about using a global name.